packages/react-utilities/src/compose/getSlots.ts

```diff
@@ -54,11 +57,11 @@ export function getSlots<SlotProps extends SlotPropsRecord = {}>(state: Componen
 
   if (typedSlotNames) {
     for (const name of typedSlotNames) {
-      const { as, children } = typedState[name];
+      const { children, ...rest } = typedState[name];
 
-      const slot = getSlot(typedState.components ? typedState.components[name] : undefined, as) as Slots[typeof name];
+      const slot = ((typedState.components && typedState.components[name]) || 'div') as Slots[typeof name];
```

Comment:
```suggestion
      const slot = (typedState.components?.[name] || 'div') as Slots[typeof name];
```

Is it the same?

Reply:
Yes, I removed optional chaining,, but can add it back